reports/update-orphaning/Update orphaning analysis using longitudinal dataset.ipynb

```diff
@@ -263,7 +266,52 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For people who are out-of-date and don't have updates disabled, determine what the update check returns."
+    "Focus on orphaned users who have updates enabled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "update_enabled_statuses = update_enabled_disabled_statuses.filter(lambda p: \"update-enabled\" in p)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For people who are out-of-date and have updates enabled, determine the distribution across Firefox versions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "def version_mapper(d):\n",
+    "    status, ping = d\n",
+    "    if ping.version[0] is None:\n",
+    "        return (\"-1\", ping)\n",
```

Comment:
I think `None` should work as well and make more sense than an arbitrary string.